packages/relay/src/ServerConfigParams.ts

```diff
@@ -368,6 +370,22 @@ export function validateBalanceParams (config: ServerConfigParams): void {
   }
 }
 
+export async function validateHub (hubFile: string, hubAddress: string, managerAddress: string, workerAddress: string, contractInteractor: ContractInteractor): Promise<void> {
+  const managerNonce = await contractInteractor.getTransactionCount(managerAddress, 'pending')
+  const workerNonce = await contractInteractor.getTransactionCount(workerAddress, 'pending')
+  if (!fs.existsSync(hubFile) || (managerNonce === 0 && workerNonce === 0)) {
+    fs.writeFileSync(hubFile, hubAddress)
+    console.log(`RelayHub lock file ${hubFile} created.\n`)
+  } else {
+    console.log(`Using existing lock file ${hubFile}.\n`)
+    const lockedHub = fs.readFileSync(hubFile, 'utf8')
+    if (lockedHub !== hubAddress) {
```

Comment:
Please use `isSameAddress` to avoid uppercase/lowercase. Also, there is no reason not to print both addresses as part of an error message, in case the error is in having an extra whitespace character in config or something.

Reply:
Whitespace is not an issue when parsing the config. I can use `isSameAddress`, though it seems unnecessary to me... why would you add it lowercase, then change it to the exact same address but checksummed or vice versa?

Reply:
> Whitespace is not an issue when parsing the config. I can use `isSameAddress`, though it seems unnecessary to me... why would you add it lowercase, then change it to the exact same address but checksummed or vice versa?

Basically, because you can... I don't know, I prefer comparing addresses taking that into account.